### ISSUE TITLE
Linux LVM volume type missing from webmin, latest builds?

### ISSUE THREAD

**Issue by extremeskillz** [webmin/webmin#1545]
Hi Dev team,
I noticed that in creating new partitions in prep for LVM use that the TYPE option is missing Linux LVM. I have all other types available but those don't work with LVM when creating a new group to include the partition. Unless I'm missing something, I would consider this a bug. See attached pic. 

Webmin: 1.981

![Capture](https://user-images.githubusercontent.com/40648180/140980004-c4abb841-1292-493c-91db-515db53e1c17.PNG)

**Comment by jcameron** [comment]
This is because when using `parted` to manage partitions, LVM isn't available as a type anymore. Instead you can just select "Linux EXT", and it will work fine in the LVM module.

**Comment by jcameron** [comment]
Actually it looks like the LVM state of a partition is a separate flag that can be enabled : https://www.linuxquestions.org/questions/linux-general-1/using-parted-command-to-create-lvm-partitions-4175533903/

**Comment by extremeskillz** [comment]
I created the Linux EXT then in the LVM manager I creaited a new group, selecting this volumn to add to the group and I get this error.
![image](https://user-images.githubusercontent.com/40648180/141158513-b6a2e19e-5aa9-4075-be29-c8674901b48f.png)

I know the partition is not mounted. What could be causing this?

**Comment by extremeskillz** [comment]
I resolved the issue. Turns out the disk in question was the issue. I removed the disk, reboot, re-added it, format with new partition table, and then a new partition with LINUX EXT. I was able to successfully created the LVM in the gui. 
No modification or errors came up either. So it could of been anything linking the disk somewhere where it wasn't showing in any of the list commands. 
This is a VM btw. So removing disks and adding was that easy and how I resolved it.
